blackrock_data_fetcher.py

```diff
@@ -29,7 +29,7 @@
 
 try:
     from local_settings import (
-        SFTP_HOST, SFTP_USER, SFTP_PASSWD,
+        SFTP_HOST, SFTP_USER, SFTP_USER, SFTP_PASSWD,
```

Comment:
Hey Nik! I think this is suppose to have SFTP_PORT instead of the other SFTP_USER